src/PowerShellEditorServices.Host/PowerShellEditorServices.Host.csproj

```diff
@@ -20,8 +20,6 @@
     <PackageReference Include="Microsoft.PowerShell.SDK">
       <Version>6.0.0-alpha13</Version>
     </PackageReference>
-    <PackageReference Include="System.Runtime.Extensions" Version="4.3.0" />
```

Comment:
How come you removed this? Is it because PowerShellEditorServices project uses it?

Reply:
Hmmmm, this might have been my "rip out redundant dependencies" reflex kicking in from the v2 refactor. Perhaps I should put it back in here?